BlogTemplate/.template.config/template.json

```diff
@@ -0,0 +1,21 @@
+{

```

Comment:
That was my attempt at using the templating feature but I thought I hadn't committed that.